packages/fiori/src/ViewSettingsDialog.hbs

```diff
@@ -68,7 +68,7 @@
 						sort-by>
 						<ui5-li-groupheader>{{_sortByLabel}}</ui5-li-groupheader>
 						{{#each _currentSettings.sortBy}}
-							<ui5-li ?selected="{{this.selected}}">{{this.text}}</ui5-li>
+							<ui5-li ?selected="{{this.selected}}" .associatedItem="{{this.item2}}">{{this.text}}</ui5-li>
```

Comment:
Is "item2" an existing field, somehow did not find it in the change?

Reply:
yes, I forgot to delete it